chain/vm/vmi.go

```diff
@@ -0,0 +1,23 @@
+package vm
+
+import (
+	"context"
+	"os"
+
+	"github.com/filecoin-project/lotus/chain/types"
+	"github.com/ipfs/go-cid"
+)
+
+type VMI interface {
+	ApplyMessage(ctx context.Context, cmsg types.ChainMsg) (*ApplyRet, error)
+	ApplyImplicitMessage(ctx context.Context, msg *types.Message) (*ApplyRet, error)
+	Flush(ctx context.Context) (cid.Cid, error)
+}
+
+func NewVM(ctx context.Context, opts *VMOpts) (VMI, error) {
+	if os.Getenv("LOTUS_USE_FVM_EXPERIMENTAL") == "1" {
+		return NewFVM(ctx, opts)
+	}
+
+	return NewLotusVM(ctx, opts)
```

Comment:
Can we call this `LegacyVM`? It's a bit more specific than `LotusVM`.

Reply:
Sure!